frontend/src/pages/AccountManagement/MyGroups.js

```diff
@@ -25,6 +29,12 @@ const mapRecipients = (recipients) => recipients.map((recipient) => ({
   options: mapGrants(recipient.grants),
 }));
 
+export const GROUP_FIELD_NAMES = {
+  NAME: 'new-group-name',
+  RECIPIENTS: 'select-recipients-new-group',
+  IS_PRIVATE: 'is-private',
```

Comment:
Should this have a bool value?

Reply:
These are the HTML names of the inputs (name="is-private")